gems/sorbet-runtime/test/types/props/decorator.rb

```diff
@@ -383,4 +383,13 @@ class MatrixStruct
     end
     assert_match(/Use of `optional: :existing` is not allowed/, e.message)
   end
+
+  it 'raises if the word secret appears in a prop' do
```

Comment:
could the name of a test contain what it actually tests?
It's unclear why `raises if the word secret appears in a prop` is a property that we should have.